src/en/space-station-14/round-flow/proposals/specter.md

```diff
@@ -0,0 +1,151 @@
+
+At any point, if the Specter gets close to someone whose fear is at the Hysterical level, the Specter can click on them to claim their soul.
+This instantly kills the victim, rewards the Specter with a soul, and increases their max power by 50.
+
+The haunt lasts for 60 seconds or until a soul is claimed.
+Afterwards, there is a minute-long cooldown before another haunt can start.
+
+## Objectives
+- Claim 7-12 souls
+- Raise 15-20 players' fear to Hysterical
+- Raise 3-5 player's fear to Terrified or higher simultaneously
+- Keep someone's fear at Terrified or higher for 3 minutes
+- Generate 3000 power through fear (excludes baseline power generation rate)
+
+## Abilities
+
+#### Tier 0 Abilities 
+Available roundstart at 0 souls.
+
+- **Whisper:**: Sends a message to a target player, slightly increasing their fear.
+- **Ectoplasm:** Secretes ectoplasm around the Specter in a small radius.
+Slipping on the ectoplasm increases fear and stuns like soap.
+- **Shudder:** Causes doors, windoors, and shutters to rapidly open and close, increasing fear for players nearby.
+- **Flicker:** Causes lights to flicker, increasing fear very slightly in a large area.
+Regular ghosts have this ability innately, allowing them to potentially aid the Specter.
+- **Graveyard Mist:** Creates clouds of smoke that obscure vision and cause rapid fear buildup.
+This is the same smoke as used for the shadow anomaly.
+- **Disruption:** Used on a target player; causes nearby items to be flung at them, causing fear and low damage on hit.
+
+#### Tier 1 Abilities
+- **Paranoia:** Causes target player to be unable to decrease fear for 1 minute.
+Additionally causes creepy popups.
+- **Animus:** Imbues a target weapon with life for 30 seconds, causing it to try and attack nearby players.
+- **Ghost Lock:** Causes all doors in radius to bolt for 10 seconds and play spooky slamming sounds.
+Afterwards, they will unbolt.
+- **The Station Bleeds:** Nearby vents, drains, and closed doors begin to seep blood, increasing fear in nearby players.
+- **Quiet:** In target radius, temporarily turns off all lights, extinguishes all flames, and mutes players.
+Players caught in the effect will gain fear over time.
+- **Possession:** The specter possesses target item for 15 seconds.
+While possessing, they can attack and cause fear to anyone they hit.
+
+#### Tier 2 Abilities
+- **Ruin:** Used on target player. 
+Causes rapid aging and drains stamina over 15 seconds.
+- **Necromancy:** Can only be used on target dead bodies.
+Causes an NPC Wandering Spirit to spawn.
+Wandering Spirits are non-aggressive but unkillable.
+They will cause fear in those nearby while present until dispelled by a Chaplain or salt.
+- **Void:** Can only be used when 5 tiles away from players.
+Creates a black void that causes an extreme spike in fear when stepped through.
+Lasts for 2.5 minutes.
+- **Shriek:** Damages windows nearby and causes lights to shatter.
```

Comment:
this is completely busted like with revenant, just find places that are unguarded and space them. if you are found move elsewhere and repeat until entire station dead. bonus points for breaking the teg windows and ashing half of engi

imo it should damage windows until theyre at say 50% destroyed and no more, so you are more spooky than just gamer murderbone #6

Reply:
i mean you can balance this effect with a decently high cost. adding onto the fact that killing people doesn't actually create any usable resource for you and murderboning turns out to literally just drain you of your resources and offer no reconciliation (whereas with rev, murderboning is 100% effective and the best way to do things)

Reply:
even if spectre doesnt need to space the entire station simply being able to is wack and would probably lead to unnecessary bwoinks